packages/ui/components/localize/src/utils/resolveLocaleConfig.js

```diff
@@ -0,0 +1,20 @@
+/**
+ *
+ * @param {string} namespace Namespace for the locale
+ * @param {any} localeDir Directory of the locale
+ * @returns Promise<LocaleConfig>
+ */
+export const resolveLocaleConfig = (namespace, localeDir) => ({
```

Comment:
Can we apply the `resolveLocaleConfig()` to all lion-ui components that have translations?

Reply:
Hi @rajkeshwar. 

Don't want to introduce a show stopper, but... keep in mind the reason for the current verbosity is to support all bundlers.

We came from a situation with oneliners like `return import('@lion/ui/combobox-translations/${myLocale}.js')`.
Rollup could handle this, Webpack couldn't.

I'm afraid that delegating the dynamic import to an external function makes it harder for all bundlers to make correct chunks.

Reply:
Hi @tlouisse,

Thank you so much for the detailed explanation and for outlining the key areas to consider. To further explore this topic, I’ve started a discussion to gather input on the direction: [#2431](https://github.com/ing-bank/lion/discussions/2431).